snappy_wrappers/wrappers/bcftools/TMB/wrapper.py

```diff
@@ -32,11 +32,11 @@
 number_snvs=$(bcftools view -R $bed_file -v snps --threads 2 -H {snakemake.input.vcf}| wc -l)
 number_indels=$(bcftools view -R $bed_file -v indels --threads 2 -H {snakemake.input.vcf}| wc -l)
 number_variants=$(bcftools view -R $bed_file --threads 2 -H {snakemake.input.vcf}| wc -l)
-number_missense_variants=$(bcftools view -R $bed_file --threads 2 -H {snakemake.input.vcf}| grep 'missense_variant'| wc -l)
+number_missense_variants=$(bcftools view -R $bed_file --threads 2 -H {snakemake.input.vcf}| grep -E '{snakemake.params.args[missense_re]}' | wc -l)
```

Comment:
Does this work when the snakemake parameter `args[missense_re]` is missing? I though that the `shell` command you complain if it tries to format the string with missing values.
I would have done the following:
- in the python part of the wrapper: `missense_re = snakemake.params.args["missense_re"] if "args" in snakemake.params.keys() and "missense_re" in snakemake.params.args.keys() else ""`
- in the bash/shell part: `if [[ -n "{missense_re}" ]] ... ` and `... grep -E '{missense_re}' ...`